Documentation/FluidTemplates/Index.rst

```diff
@@ -214,7 +214,7 @@ web developer is required to make. Simply learn and follow the conventions
 (e.g. that the path should be :file:`Resources/Private/Templates/Page/`) and the
 development will be smooth, easy and straight forward. In addition, if another
 web developer (e.g. one of your colleagues) looks at your sitepackage
-extension, he/she knows the locations and naming of files. This reduces
+extension, they know the locations and naming of files. This reduces
```

Comment:
With `they` we should use `other web developers`.

Reply:
Well, that would be a repetition, as the sentence already mentions "another web developer". Wouldn't it? What do you think? I find it a better style to only refer to them as "they", instead of repeating it. But I'll follow you.
The intention suggesting this change was to be a bit more gender-neutral (https://en.wikipedia.org/wiki/Singular_they).

Reply:
Oh I misread your comment, sorry! I read "instead of" :-)
Well as the mentioned wikipedia article says, you use the singular "they" instead of he/she to be more gender-neutral.

Reply:
I get what you mean and understand that it's actually correct. Still feels weird but then again I'm not a native English speaker. ^^